app/views/tasks/_completeness_checklist.html.slim

```diff
@@ -0,0 +1,19 @@
+.sticky-bottom
+  .completeness-checklist-container.my-3
+    .checklist-content
+      .maximized-checklist.p-3
+        h5 = t('.checklist')
```

Comment:
Maybe, depending on how the chevron is shown, we could display this "headline" for the collapsed version sideways (e.g., using CSS `transform: rotate(-90deg)`)?

Reply:
That's true, and your solution with the chevron looks pretty clean. Let's keep that :+1:.